bigflow/scaffold/scaffold_templates.py

```diff
@@ -385,8 +385,8 @@ class FakeFile(object):
 # Pyre type checker
 .pyre/
 .dags
-image
+.image
 .dags/
-image/
+.image/
```

Comment:
Maybe '/image/' (don't ignore 'image' directory/file outside the project root)
Also I don't think there is need to add both 'image' and 'image/'

Reply:
true, done